### PR TITLE
updated doc format for Running ASTRA-sim

### PR DESCRIPTION
**Pull request by zli669**

### CODE REVIEW

File: pages/docs/Argument ${NETWORK_CONFIG}.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Argument ${NETWORK_CONFIG}
```
Comment (willjwon) [inline pages/docs/Argument ${NETWORK_CONFIG}.rst:1]: All updates looks good to me, but can you please change the filenames of those `rst` to not have any special characters or spaces? These filename seems irregular and might look awkward when appeared in URLs.

Something like `network_config_arguments.rst` or whatnot.

**Review by willjwon — CHANGES_REQUESTED** [review]

**Review by zli669 — COMMENTED** [review]
@willjwon Hi Will, thank you for the comments. I have updated the file names to avoid spaces and special characters. Please review. Thanks again!

**Review by willjwon — APPROVED** [review]

**Comment by willjwon** [comment]
@zli669 Approved, thanks!

**Comment by willjwon** [comment]
Closing this PR, please migrate the contents to this repo: https://github.com/astra-sim/astra-sim-docs

**Pull request closed without merging**